src/test/jtx/impl/Env.cpp

```diff
@@ -537,6 +537,14 @@ Env::do_rpc(std::vector<std::string> const& args)
     return response;
 }
 
+void
+Env::enableFeature(uint256 const feature)
```

Comment:
you could make it a free function `enableFeature(Env& e, uint256 const f)`, which might be slightly cleaner. I do feel like this particular change should be coordinated with the pending PR I mentioned above because that changes the `Env` feature handling considerably. If you are adding `enableFeature`, does it also make sense to add a `hasFeature` function?   https://github.com/ripple/rippled/pull/2108/files#diff-4d56022e39f0891ca82a12c0ef40386bR660